tests/database_api_tests.py

```diff
@@ -252,6 +252,24 @@ def test_schemas(self):
         response = json.loads(rv.data.decode("utf-8"))
         self.assertEqual(0, len(response["result"]))
 
+    def test_database_schemas(self):
```

Comment:
suggestion: there are some complexity due to the api pagination, it may be useful to test it as well
e.g. @nytai helped me to figure out that https://github.com/dropbox/incubator-superset-internal/commit/a7aec30301a363df96f9d723ba9a36d7894f5012 setting max_page_size = -1 and page_size = -1 breaks some assumptions about pagination apis & it's not covered by unit test.

Reply:
This endpoint has no pagination, since if I'm not mistaken the dropdown does not support it yet

Reply:
We can convert the dropdown to use pagination if we think it would be useful. Currently, it only makes one api call to retrieve the list of options.

Reply:
It would be super useful on the SQLLab database dropdown